package.json

```diff
@@ -18,6 +19,7 @@
     "gatsby-transformer-remark": "^2.1.17",
     "gatsby-transformer-sharp": "^2.1.8",
     "lodash": "^4.17.11",
+    "moment-timezone": "^0.5.23",
```

Comment:
`moment-timezone` is not used anymore. Can consider uninstalling the package.

Reply:
uninstalled.